src/services/dataSrvc.js

```diff
@@ -38,5 +38,16 @@ angular.module('app').factory('DataSrvc', ['$http', '$q', '$location', function
           })
 
       },
+      getUrlData: function (url, cb) {
```

Comment:
[Actually it is generic enough that's why I named it getUrlData.] 
In this case it's actually not a _constant_ but rather a function of the url used in the getData (we should name it getAllData). The hardcoded url is a bad hack to see it working. I'll update it to use not-constant value in the next step.